src/tests/ftest/container/per_server_fault_domain.py

```diff
@@ -97,7 +96,8 @@ def test_rf1_healthy(self):
         :avocado: tags=per_server_fault_domain,rf1_healthy
         """
         # 1. Determine the ranks to stop; two ranks in the same node.
-        random_host = random.choice(list(set(self.server_managers[0].ranks.values())))
+        # Arbitrarily select one host.
+        random_host = list(set(self.server_managers[0].ranks.values()))[0]
```

Comment:
FYI, to use `random.choice`, you just need this comment:
```suggestion
        random_host = random.choice(list(set(self.server_managers[0].ranks.values())))  # nosec
```

Reply:
Thanks. I don't see much benefit of using `random.chice` here, so I'll keep it as is.

Reply:
Dalton is (nearly) correct, if you're trying to arbitrarily select one host as the comment says then random.choice is the correct way to do this, however you also don't need the list() or set() calls if doing that so I think you want the following.
```suggestion
        # Arbitrarily select one host.
        random_host = random.choice(self.server_managers[0].ranks.values())  # nosec
```

Reply:
The selection of the host is irrelevant to what we want to test. I changed it to select index 0.